.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -207,7 +207,7 @@ steps:
       echo MODE=$(MODE) && \
       if [[ "${GEN2_SIG_FOR_PRODUCTION}" == "True" ]]; then gen2_captured_sig_version="$(cat vhdbuilder/packer/settings.json | grep "gen2_captured_sig_version" | awk -F':' '{print $2}' | awk -F'"' '{print $2}')"; \
       [ -n "${gen2_captured_sig_version}" ] && VHD_NAME="${gen2_captured_sig_version}.vhd"; \
-      SKU_NAME="windows-$WINDOWS_SKU"; \
+      SKU_NAME="windows-$WINDOWS_SKU-gen2"; \
```

Comment:
I think the easiest way to fix is the one you are commenting here (the first commit). It requires only the `SKU_NAME` to have "gen2" suffix and then be outputted to the published artifact `vhd-publishing-info.json`. However, this is inconsistent with the one defined in line 119 above.

The better way may be defining a new `WINDOWS_SKU` in line 119 since logically there should be a new SKU for V2? This way, the names are more consistent. "gen2" will be included in `WINDOWS_SKU`, `VNET_RESOURCE_GROUP_NAME`, `artifactName` etc., basically everywhere involving Gen 2?